src/transaction/boot_cl.c

```diff
@@ -5455,29 +5631,29 @@ catcls_vclass_install (void)
   // *INDENT-OFF*
   struct catcls_function
   {
-    const char *name;
     const DEF_FUNCTION function;
   }
   clist[] =
   {
-    {"CTV_CLASS_NAME", boot_define_view_class},
-    {"CTV_SUPER_CLASS_NAME", boot_define_view_super_class},
-    {"CTV_VCLASS_NAME", boot_define_view_vclass},
-    {"CTV_ATTRIBUTE_NAME", boot_define_view_attribute},
-    {"CTV_ATTR_SD_NAME", boot_define_view_attribute_set_domain},
-    {"CTV_METHOD_NAME", boot_define_view_method},
-    {"CTV_METHARG_NAME", boot_define_view_method_argument},
-    {"CTV_METHARG_SD_NAME", boot_define_view_method_argument_set_domain},
-    {"CTV_METHFILE_NAME", boot_define_view_method_file},
-    {"CTV_INDEX_NAME", boot_define_view_index},
-    {"CTV_INDEXKEY_NAME", boot_define_view_index_key},
-    {"CTV_AUTH_NAME", boot_define_view_authorization},
-    {"CTV_TRIGGER_NAME", boot_define_view_trigger},
-    {"CTV_PARTITION_NAME", boot_define_view_partition},
-    {"CTV_STORED_PROC_NAME", boot_define_view_stored_procedure},
-    {"CTV_STORED_PROC_ARGS_NAME", boot_define_view_stored_procedure_arguments},
-    {"CTV_DB_COLLATION_NAME", boot_define_view_db_collation},
-    {"CTV_DB_CHARSET_NAME", boot_define_view_db_charset}
+    { boot_define_view_class},  /* CTV_CLASS_NAME */
+    { boot_define_view_super_class}, /* CTV_SUPER_CLASS_NAME */
+    { boot_define_view_vclass}, /* CTV_VCLASS_NAME */
+    { boot_define_view_attribute}, /* CTV_ATTRIBUTE_NAME */
+    { boot_define_view_attribute_set_domain}, /* CTV_ATTR_SD_NAME */
+    { boot_define_view_method}, /* CTV_METHOD_NAME */
+    { boot_define_view_method_argument}, /* CTV_METHARG_NAME */
+    { boot_define_view_method_argument_set_domain}, /* CTV_METHARG_SD_NAME */
+    { boot_define_view_method_file}, /* CTV_METHFILE_NAME */
+    { boot_define_view_index}, /* CTV_INDEX_NAME */
+    { boot_define_view_index_key}, /* CTV_INDEXKEY_NAME */
+    { boot_define_view_authorization}, /* CTV_AUTH_NAME */
+    { boot_define_view_trigger}, /* CTV_TRIGGER_NAME */
+    { boot_define_view_partition}, /* CTV_PARTITION_NAME */
+    { boot_define_view_stored_procedure}, /* CTV_STORED_PROC_NAME */
+    { boot_define_view_stored_procedure_arguments}, /* CTV_STORED_PROC_ARGS_NAME */
+    { boot_define_view_db_collation}, /* CTV_DB_COLLATION_NAME */
+    { boot_define_view_db_charset}, /* CTV_DB_CHARSET_NAME */
+    { boot_define_view_db_server} /* CTV_DB_SERVER_NAME */
```

Comment:
This refactoring seems not the scope of this PR.

Reply:
The original code was as below.
{**"CTV_CLASS_NAME"**, boot_define_view_class},

If corrected correctly, the quotes should be removed as shown below.
     { **CTV_CLASS_NAME**, boot_define_view_class},
However, the name member variable of the catcls_function structure was never used.

Even if I modify it correctly, it only takes up unnecessary memory, so I modified it because there was an opinion that it would be better to remove it altogether.

Since there are no other influences, I don't think there is a need to make it a separate issue. Would it be better to treat it as a separate issue though?

Reply:
I couldn't recognize that this structure's scope is in the `catcls_vclass_install()` function. yes, It seems no impact.

Reply:
Because of `// *INDENT OFF*` hint, formatting seems broken.
```suggestion
    {boot_define_view_class},  /* CTV_CLASS_NAME */
    {boot_define_view_super_class}, /* CTV_SUPER_CLASS_NAME */
    {boot_define_view_vclass}, /* CTV_VCLASS_NAME */
    {boot_define_view_attribute}, /* CTV_ATTRIBUTE_NAME */
    {boot_define_view_attribute_set_domain}, /* CTV_ATTR_SD_NAME */
    {boot_define_view_method}, /* CTV_METHOD_NAME */
    {boot_define_view_method_argument}, /* CTV_METHARG_NAME */
    {boot_define_view_method_argument_set_domain}, /* CTV_METHARG_SD_NAME */
    {boot_define_view_method_file}, /* CTV_METHFILE_NAME */
    {boot_define_view_index}, /* CTV_INDEX_NAME */
    {boot_define_view_index_key}, /* CTV_INDEXKEY_NAME */
    {boot_define_view_authorization}, /* CTV_AUTH_NAME */
    {boot_define_view_trigger}, /* CTV_TRIGGER_NAME */
    {boot_define_view_partition}, /* CTV_PARTITION_NAME */
    {boot_define_view_stored_procedure}, /* CTV_STORED_PROC_NAME */
    {boot_define_view_stored_procedure_arguments}, /* CTV_STORED_PROC_ARGS_NAME */
    {boot_define_view_db_collation}, /* CTV_DB_COLLATION_NAME */
    {boot_define_view_db_charset}, /* CTV_DB_CHARSET_NAME */
    {boot_define_view_db_server} /* CTV_DB_SERVER_NAME */
```

Reply:
thank you, I will clear white space.